src/component/tooltip/TooltipHTMLContent.ts

```diff
@@ -60,17 +60,21 @@ function mirrorPos(pos: string): string {
 }
 
 function assembleArrow(
-    backgroundColor: ColorString,
+    tooltipModel: Model<TooltipOption>,
     borderColor: ZRColor,
     arrowPosition: TooltipOption['position']
 ) {
     if (!isString(arrowPosition) || arrowPosition === 'inside') {
         return '';
     }
 
+    const backgroundColor = tooltipModel.get('backgroundColor');
+    const borderWidth = tooltipModel.get('borderWidth');
+
     borderColor = convertToColorString(borderColor);
     const arrowPos = mirrorPos(arrowPosition);
-    let positionStyle = `${arrowPos}:-6px;`;
+    const arrowSize = borderWidth >= 5 ? borderWidth : 5;
```

Comment:
I think we're not modifying `borderWidth`, just the `size of the arrow` if borderWidth is less than 5 because below this the arrow is very very small and looking weird.
Do you think I should remove it?

Reply:
I think so. Because it will look weird if we set a small `borderWidth`, for example, `1`.

<img src="https://user-images.githubusercontent.com/26999792/118921914-9e18d980-b96b-11eb-8ea9-45882264df88.png" width="400">

Reply:
So, for smaller borderwidth I'll handle position in an `if` condition. Is this fine?

Reply:
Can have a try.

Reply:
:heavy_check_mark: